.bumpversion.cfg

```diff
@@ -0,0 +1,6 @@
+[bumpversion]
+parse = (?P<major>\d+)\.(?P<minor>\d+)\.(?P<patch>\d+)|dev
+
+[bumpversion:file:setup.py]
+[bumpversion:file:u8timeseries/__init__.py]
+[bumpversion:file:conda_recipe/u8timeseries/meta.yaml]
```

Comment:
can you also add `u8timeseries/docs/source/conf.py` as mentioned by julien? think it doesn't get updated otherwise

Reply:
Good point, adding it